entrypoint.sh

```diff
@@ -74,8 +74,12 @@ parse_args() {
   return 0
 }
 
-install_ansible_lint() {
-  pip install ansible-lint=="${VERSION:-${DEFAULT_VERSION}}" || exit 1
+override_python_packages() {
+  echo "Overrides: ${OVERRIDE}"
```

Comment:
Other `echo`s in this script write to stderr
```suggestion
  >&2 echo "Overrides: ${OVERRIDE}"
```